.github/workflows/test.yml

```diff
@@ -128,6 +136,9 @@ jobs:
                 coverage: xdebug
           - name: Checkout Nextcloud
             run: git clone https://github.com/nextcloud/server.git --recursive --depth 1 -b ${{ matrix.nextcloud-versions }} nextcloud
+          - name: Patch version check for nightly PHP
+                  if: ${{ matrix.php-versions == '8.2' }}
```

Comment:
That looks off ;)

Reply:
While fixing it squash the commits right away. I think CI should pass now